build.gradle

```diff
@@ -122,6 +128,17 @@ def signingKey(project) {
       }
     } // signingConfigs
 
+    playAccountConfigs {
+      defaultAccountConfig {
+        serviceAccountEmail = 'name@server.com'
+        pk12File = project.file('key.p12') // Will need to encrypt this in Travis, obvs
```

Comment:
Are you able to clarify what this stuff is about, and what I need to do (or what privilege you would need to have added if that's possible)?

(Sorry for my lack of knowledge of this Android publishing side - it seems to change constantly!)